src/components/HLSPlayback.tsx

```diff
@@ -1,33 +1,34 @@
 import { useCallback, useRef, useState } from "react";
 import Hls from "hls.js";
+import { useServerSdk } from "./ServerSdkContext";
+import { CopyLinkButton } from "./CopyButton";
 
-const BIG_BUCK_BUNNY_SRC = "https://test-streams.mux.dev/x36xhzz/x36xhzz.m3u8";
+export default function HlsPlayback({ roomId, isPlayable }: { roomId: string; isPlayable: boolean }) {
+  const { signalingHost } = useServerSdk();
+  const [autoPlay, setAutoPlay] = useState<boolean>(false);
 
-export default function HlsPlayback() {
   const hls = useRef<Hls | null>(null);
-  const [src, setSrc] = useState(BIG_BUCK_BUNNY_SRC);
-  const [srcInput, setSrcInput] = useState(src);
-
+  const hlsLink = `http://${signalingHost}/hls/${roomId}/index.m3u8`;
```

Comment:
You need to use http or https from context because it could be secure connection